ts/Extensions/Annotations/Types/BasicAnnotation.ts

```diff
@@ -348,6 +348,7 @@ class BasicAnnotation extends Annotation {
         if (options.shapes) {
             delete options.labelOptions;
             const type = options.shapes[0].type;
+            options.shapes[0].className = 'highcharts-basic-shape';
```

Comment:
This will replace user-defined classnames. We should concat: `classname += 'highcharts-basic-shape'`

With advanced annotations it is fine, because users don't have direct access to each shape (IIRC)